node/utils/regionV1.ts

```diff
@@ -0,0 +1,7 @@
+export function isRegionV1(regionId: string) {
+  return !regionId.includes('v2')
```

Comment:
This code should take care future versions? If yes, maybe would be better use a regex to find "v" followed by some number, like this:

```suggestion
  const regex = /v\d+/; 
  return !regex.test(regionId);
```

Reply:
Nice! Agree with you!